crates/aptos/src/test/mod.rs

```diff
@@ -274,13 +291,17 @@ impl CliTestFramework {
         .await
     }
 
-    pub async fn init(&self, private_key: &Ed25519PrivateKey) -> CliTypedResult<()> {
+    pub async fn init(
+        &self,
+        profile: String,
+        private_key: &Ed25519PrivateKey,
+    ) -> CliTypedResult<()> {
         InitTool {
             rest_url: Some(self.endpoint.clone()),
             faucet_url: Some(self.faucet_endpoint.clone()),
             rng_args: RngArgs::from_seed([0; 32]),
             private_key_options: PrivateKeyInputOptions::from_private_key(private_key)?,
-            profile_options: Default::default(),
+            profile_options: ProfileOptions { profile },
```

Comment:
Remove the profiles, they were a cause of flakiness in the tests.  Should be back to default.

Reply:
this was actually intentional, but let me leave that for a separate diff.